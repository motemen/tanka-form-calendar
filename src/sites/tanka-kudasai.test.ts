import 短歌ください from "./tanka-kudasai";

describe("tanka-kudasai", () => {
  it("crawl", async () => {
    const events = await 短歌ください.crawl();
    expect(events).toMatchSnapshot();
  });
});
