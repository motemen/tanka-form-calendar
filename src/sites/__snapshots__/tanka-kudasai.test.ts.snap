// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tanka-kudasai crawl 1`] = `
Array [
  Object {
    "date": Array [
      2022,
      6,
      30,
    ],
    "detail": Object {
      "theme": "ぴょんぴょん",
    },
    "key": "7b83cf9e07e56e588064761b5eec978a",
  },
]
`;
